packages/manager/apps/web-hosting/src/components/breadcrumb/Breadcrumb.component.tsx

```diff
@@ -19,6 +20,11 @@ export const Breadcrumb: React.FC<{ namespace?: string | string[] }> = (
   const { t } = useTranslation(namespace);
   const matches = useMatches();
 
+  const rootItem = {
```

Comment:
I don't understand why you have to add this ? Can't you manage it from your route definition ?